README.md

```diff
@@ -27,13 +27,14 @@ If you want to use this dataset in your ESMValTool projects, add the following l
 
 ```yaml
 rootpath:
-  CMIP6: <path_to_repository>/esmvaltool_sample_data/data/timeseries/
+  CMIP6:
+  - <path_to_repository>/esmvaltool_sample_data/data/timeseries/CMIP6
 
 drs:
   CMIP6: default
 ```
 
-If you have installed `esmvaltool_sample_data`, you can find the `rootpath` using `python -c 'from esmvaltool_sample_data import base_dir; print(base_dir.parent)'`
+If you have installed `esmvaltool_sample_data`, you can find the `rootpath` using `python -c 'import esmvaltool_sample_data, yaml; print(yaml.dump(esmvaltool_sample_data.get_rootpaths()))'`
```

Comment:
Why is `yaml` needed to print a python object? Is it for pretty printing? In that case it might make more sense to use [`pprint`](https://docs.python.org/3/library/pprint.html#pprint.pprint) from the standard library.

Reply:
I wanted to have it so that you can copy/paste the output into the `config-user.yml` directly. `pprint` does not do that, it just prints the dict.

Reply:
OK, maybe mention that in the text then?